runtime/queries/metricsview.go

```diff
@@ -404,3 +409,107 @@ func writeXLSX(meta []*runtimev1.MetricsViewColumn, data []*structpb.Struct, wri
 
 	return err
 }
+
+func writeParquet(meta []*runtimev1.MetricsViewColumn, data []*structpb.Struct, ioWriter io.Writer) error {
+	fields := make([]arrow.Field, 0, len(meta))
+	for _, f := range meta {
+		arrowField := arrow.Field{}
+		arrowField.Name = f.Name
+		switch f.Type {
+		case "CODE_BOOL":
+			arrowField.Type = arrow.FixedWidthTypes.Boolean
+		case "CODE_INT8":
+			arrowField.Type = arrow.PrimitiveTypes.Int8
+		case "CODE_INT16":
+			arrowField.Type = arrow.PrimitiveTypes.Int16
+		case "CODE_INT32":
+			arrowField.Type = arrow.PrimitiveTypes.Int32
+		case "CODE_INT64":
+			arrowField.Type = arrow.PrimitiveTypes.Int64
+		case "CODE_INT128":
+			arrowField.Type = arrow.PrimitiveTypes.Float64
+		case "CODE_UINT8":
+			arrowField.Type = arrow.PrimitiveTypes.Uint8
+		case "CODE_UINT16":
+			arrowField.Type = arrow.PrimitiveTypes.Uint16
+		case "CODE_UINT32":
+			arrowField.Type = arrow.PrimitiveTypes.Uint32
+		case "CODE_UINT64":
+			arrowField.Type = arrow.PrimitiveTypes.Uint64
+		case "CODE_DECIMAL":
+			arrowField.Type = arrow.PrimitiveTypes.Float64
+		case "CODE_FLOAT32":
+			arrowField.Type = arrow.PrimitiveTypes.Float32
+		case "CODE_FLOAT64":
+			arrowField.Type = arrow.PrimitiveTypes.Float64
+		case "CODE_STRUCT", "CODE_UUID", "CODE_ARRAY", "CODE_TIMESTAMP", "CODE_TIME", "CODE_DATE", "CODE_STRING", "CODE_MAP":
+			arrowField.Type = arrow.BinaryTypes.String
```

Comment:
Timestamps are very common, so we should use the arrow type for that (it might be an annotated type, don't remember)

Reply:
That depends on this https://github.com/xitongsys/parquet-go/pull/551